test/test_common/status_helpers.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
nit: This file is a dup of status_utility.h

Reply:
It looks like a botched a rename.  Fixed!